### PR TITLE
fix a bug in evaluating accuracy

### PR DESCRIPTION
**Pull request by llcao**
Correct me if I am wrong: I think the code in the current example:

```
def compute_accuracy(predictions, labels):
    '''Compute classification accuracy with a fixed threshold on distances.
    '''
    return labels[predictions.ravel() < 0.5].mean()
```

is not accuracy over all the samples, but over samples with negative prediction.

### CODE REVIEW

**Comment by fchollet** [comment]
?

**Comment by llcao** [comment]
sorry I was not clear. to show this, let's consider two examples:
```
def compute_accuracy(predictions, labels):
    # orgin code
    return labels[predictions.ravel() < 0.5].mean()

print compute_accuracy(np.array([0.6, 0.5]), np.array([1,1])) # result is nan, but should be 1.0

print compute_accuracy(np.array([0.6, 0.4]), np.array([0,1])) # result is 1.0, but should be 0.5
```

File: examples/mnist_siamese_graph.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def create_base_network(input_dim):
 def compute_accuracy(predictions, labels):
     '''Compute classification accuracy with a fixed threshold on distances.
     '''
-    return labels[predictions.ravel() < 0.5].mean()
+    return np.mean(labels==(predictions.ravel() > 0.5))
```
Comment (fchollet) [inline examples/mnist_siamese_graph.py]: PEP8: put spaces around `==`. Otherwise LGTM.

Comment (llcao) [inline examples/mnist_siamese_graph.py]: Done. Thanks for the suggestion.

**Pull request merged**